src/components/Branding/branding.css

```diff
@@ -30,15 +30,15 @@
 
 @screen sm {
   .lil-brand-text {
-    bottom: -4rem;
+    bottom: -2rem;
```

Comment:
verified for all screens ?

Reply:
yes, only changes spacing for branding text